Extension/lib/utils/filters-storage.js

```diff
@@ -0,0 +1,93 @@
+/* global require, exports */
+/**
+ * This file is part of Adguard Browser Extension (https://github.com/AdguardTeam/AdguardBrowserExtension).
+ *
+ * Adguard Browser Extension is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Adguard Browser Extension is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Adguard Browser Extension.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+var Log = require('../../lib/utils/log').Log; // jshint ignore:line
+var LS = require('../../lib/utils/local-storage').LS;
+
+/**
+ * Helper class for working with filters metadata storage (local storage)
+ */
+var FilterLSUtils = exports.FilterLSUtils = {
```

Comment:
because of the tests, it's in antibanner.js now, and I don't want to link it all.